portal/src/FormContainer.tsx

```diff
@@ -117,18 +76,46 @@ const FormContainer_: React.VFC<FormContainerProps> = function FormContainer_(
 
   return (
     <>
-      <CommandBarContainer
-        renderHeaderContent={renderHeaderContent}
-        hideCommandBar={hideCommandBar}
-        isLoading={isUpdating}
-        primaryItems={items}
-        secondaryItems={farItems}
+      <DefaultLayout
+        position={stickyFooterComponent ? "sticky" : "end"}
+        HeaderComponent={HeaderComponent}
+        isLoading={false}
```

Comment:
The only use of DefaultLayout is here, and EditPictureScreen. Does EditPictureScreen need to use `isLoading`? If not, then we should remove isLoading from DefaultLayout.